cmd/mgo/main.go

```diff
@@ -28,7 +29,14 @@ func initDB() model.Router {
 		appStorage,
 		userStorage,
 	)
-	r := ihttp.NewRouter(nil, appStorage, userStorage, tokenStorage, tokenService)
+
+	corsOptions := cors.Options{
```

Comment:
CORS should be optional and disabled by default

Reply:
There is a check for nil at the 36th line in `api_router.go`